### ISSUE TITLE
Merging of the official branch and cuu branch

### ISSUE THREAD

**Issue by Godzil** [clockworkpi/launcher#31]
Hi,

Could you use "rebase" instead of merging between the cuu branch and the main branch? It make the repository history really clunky and hard to follow.
It is too late for the existing history, but could you please use the rebase strategy for future pull requests?

Thanks

(that makes merging on my branch unnecessarily complicated)

**Comment by cuu** [comment]
Rebase method is not suitable for multi-person collaborative work
so closing  this issue  now
